src/screens/HomePage/HomePage.js

```diff
@@ -115,10 +102,13 @@ export default function HomePage() {
                   justify="flex-end"
                 >
                   <SelectorWrapper>
-                    <GlobalSelector
-                      handleChange={handleSelectChange}
-                      extendRight={undefined}
-                    />
+                    <ParentSize>
+                      {() => (
+                        <SearchAutocomplete
+                          locations={getSearchAutocompleteLocations()}
+                        />
+                      )}
+                    </ParentSize>
```

Comment:
We might not need `ParentSize`, we should be able to set the styles so it expands to take the width of its parent. Using `ParentSize` should be used when we _really_ need the size in pixels, because `ParentSize` needs to render a couple of times to get the size in pixels

Reply:
going to punt this for after merging it into msa preview link (as we discussed- still gotta work on properly sizing it on mobile)